lib/Helpers/Mail/Attachment.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * This helper builds the request body for a /mail/send API call.
+ *
+ * PHP version 5.6, 7
+ *
+ * @author    Elmer Thomas <dx@sendgrid.com>
+ * @copyright 2017 SendGrid
+ * @license   https://opensource.org/licenses/MIT The MIT License
+ * @version   GIT: <git_id>
+ * @link      http://packagist.org/packages/sendgrid/sendgrid
+ */
+
+namespace SendGrid\Mail;
+
+class Attachment implements \JsonSerializable
+{
```

Comment:
If an `Attachment` is needed, what's the point of having setters for the mandatory properties such as `content, type, filename, disposition`?, I would leave only one `setter` which is `id`, the rest should be mandatory.